CSharp/Library/Microsoft.Bot.Connector.NetCore/Microsoft.Bot.Connector.NetCore.nuget.props

```diff
@@ -5,7 +5,7 @@
     <RestoreTool Condition=" '$(RestoreTool)' == '' ">NuGet</RestoreTool>
     <ProjectAssetsFile Condition=" '$(ProjectAssetsFile)' == '' ">C:\source\github\BotBuilder-V3\CSharp\Library\Microsoft.Bot.Connector.NetCore\project.lock.json</ProjectAssetsFile>
     <NuGetPackageRoot Condition=" '$(NuGetPackageRoot)' == '' ">$(UserProfile)\.nuget\packages\</NuGetPackageRoot>
-    <NuGetPackageFolders Condition=" '$(NuGetPackageFolders)' == '' ">C:\Users\tomlm\.nuget\packages\</NuGetPackageFolders>
+    <NuGetPackageFolders Condition=" '$(NuGetPackageFolders)' == '' ">C:\Users\ccastro\.nuget\packages\</NuGetPackageFolders>
```

Comment:
Is this is a build-break waiting to happen?

Reply:
Ha, good point. I'll look into it.

Reply:
Fixed with a build variable.